DEVELOPING.md

```diff
@@ -354,6 +354,40 @@ stacktrace that was saved as a txt file.
 $ ndk-stack -sym obj/arm64-v8a -dump trace.txt
 ```
 
+## Release management
+
+The Mapbox Maps SDK for Android makes stable releases every 6 weeks. Between releases, updates are available as snapshots before the issuance of a beta build in week 2 and a release candidate in week 4, which is promoted to stable in week 6.
+
+### What is a snapshot?
+A snapshot release is a mechanism by which the latest commits are available as a build. Working with snapshots is described in detail in a section below.
+
+### Why do we provide pre-releases?
+We provide pre-releases during development to enable the community and our customers to provide feedback.
+
+### What is the beta release?
+A beta release is a pre-release that exposes initial APIs and behaviors of upcoming features for early feedback. This release is made during week 2 of the release cadence.
+
+### What is the RC release?
+The scope of this beta release is finalized during week 4 of the cadence, and the release branch is created. Once the release branch is cut, we normally will not introduce any new APIs, and the RC is then promoted to stable in week 6.
```

Comment:
Is it true regarding the branch though?

Reply:
I mean - we cut release branch before releasing beta, not rc

Reply:
goot catch, should be moved in section above for the beta release